pkg/guacrest/server/server.go

```diff
@@ -99,3 +99,63 @@ func (s *DefaultServer) AnalyzeDependencies(ctx context.Context, request gen.Ana
 		return nil, fmt.Errorf("%v sort is unsupported", request.Params.Sort)
 	}
 }
+
+func (s *DefaultServer) GetPackagePurls(ctx context.Context, request gen.GetPackagePurlsRequestObject) (gen.GetPackagePurlsResponseObject, error) {
+	return gen.GetPackagePurls200JSONResponse{}, nil
+}
+
+func (s *DefaultServer) GetPackageVulns(ctx context.Context, request gen.GetPackageVulnsRequestObject) (gen.GetPackageVulnsResponseObject, error) {
+	return gen.GetPackageVulns200JSONResponse{}, nil
+}
+
+func (s *DefaultServer) GetPackageDeps(ctx context.Context, request gen.GetPackageDepsRequestObject) (gen.GetPackageDepsResponseObject, error) {
+	purls, err := GetDepsForPackage(ctx, s.gqlClient, request.Purl)
+	if err != nil {
+		err, ok := handleErr(ctx, err, GetPackageDeps).(gen.GetPackageDepsResponseObject)
```

Comment:
`handleErr` shouldn't return an error. If it failed, it itself should successfully return the `gen.GetPackageDeps400JSONResponse` object explaining the failure.  (here and below)

Reply:
Done, thanks